code/controllers/subsystem/garbage.dm

```diff
@@ -284,12 +284,6 @@ SUBSYSTEM_DEF(garbage)
 // Should be treated as a replacement for the 'del' keyword.
 // Datums passed to this will be given a chance to clean up references to allow the GC to collect them.
 /proc/qdel(datum/D, force=FALSE, ...)
-	if(isweakref(D))
-		var/datum/weakref/weakref = D
-		D = weakref.resolve()
-		if(!D)
-			return
```

Comment:
I really hope this doesn't break anything.
What's the reason for this change?

Reply:
Literally just got moved to `/datum/weakref/Destroy(force)` so it stops typechecking every datum